apps/spruce/src/components/Content/Layout.tsx

```diff
@@ -10,22 +9,14 @@ import { Feedback } from "components/Feedback";
 import { Header } from "components/Header";
 import { SiteLayout } from "components/styles/Layout";
 import { TaskStatusIconLegend } from "components/TaskStatusIconLegend";
-import WelcomeModal from "components/WelcomeModal";
-import { CY_DISABLE_NEW_USER_WELCOME_MODAL } from "constants/cookies";
-import { newSpruceUser } from "constants/welcomeModalProps";
 import { useAuthStateContext } from "context/Auth";
 import { UserQuery, UserQueryVariables } from "gql/generated/types";
 import { USER } from "gql/queries";
-import { useUserSettings } from "hooks";
 import { useAnnouncementToast } from "hooks/useAnnouncementToast";
 import { WaterfallScrollToTop } from "pages/waterfall/ScrollToTop";
-import { isProduction } from "utils/environmentVariables";
 
 const { gray, white } = palette;
 
-const shouldDisableForTest =
-  !isProduction() && Cookies.get(CY_DISABLE_NEW_USER_WELCOME_MODAL) === "true";
```

Comment:
should we delete this cookie entirely or do we expect to use it again?

Reply:
It's used to disable the modal in testing, I opted not to rename because we will be able to delete it entirely when we remove the modal. I could though!

Reply:
I think that's true for `CY_DISABLE_COMMITS_WELCOME_MODAL`, not sure for `CY_DISABLE_NEW_USER_WELCOME_MODAL` which seems unused